packages/v4/src/content/design-guidelines/components/drag-and-drop/drag.md

```diff
@@ -0,0 +1,72 @@
+---
+id: Drag and drop
+section: components
+---
+The **drag and drop** interaction can be used to reposition elements on screen into a layout that benefits the user. This gives the user more flexibility to arrange and/or group items without having to make code changes.
+
+## Usage
+### Simple list
+1. **Draggable area:** The drag and drop interaction can be triggered with click and hold over any part of the item. The `fa-grip` icon is used to show that the drag and drop interaction is available.
+
+<img src="./img/Simple-list-drag-1.png" width="588" alt="Showing the area in which an item can be dragged."/>
+
+2. **Bounding box:** Upon click & hold a `pf-color-blue-400` border will show the draggable area that is available. 
+
+3. **onDrag event:** The list item being dragged will also use a `pf-color-blue-400` border to highlight it as the item being dragged and all other list items will switch to a disabled state. The space where the item is being dragged from will remain empty to indicate its original position in the list.
+
+<img src="./img/Simple-list-drag-2-3.png" width="683" alt="Borders appear around the item and bounding box shows the area in which an item can be dragged."/>
+
+4. **postDrag event:** Once dropped, the items will be reordered based on the user’s action. The space left empty is then filled by the next item in the list.
```

Comment:
On the event names, do we want to give the event name or a more descriptive phrase? And if we use the event name, do we want to try and pair with the HTML drag/drop API or something else? https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API